README.md

```diff
@@ -237,6 +237,8 @@ useDefault = true
 # or you can supply a path to a configuration. Path is relative to where gitleaks
 # was invoked, not the location of the base config.
 path = "common_config.toml"
+# If there are any rules you don't want to inherit, they can be specified here.
+ignoredRules = [ "awsome-rule-2"]
```

Comment:
```suggestion
ignoredRules = [ "generic-api-key"]
```

I suggest adding `generic-api-key` from the default config here, as a lot of folks might want to disable this due to the amount of false-positives.